test/ValidateID.cpp

```diff
@@ -406,7 +406,7 @@ class OpTypeArrayLengthTest
     : public spvtest::TextToBinaryTestBase<::testing::TestWithParam<int>> {
  protected:
   OpTypeArrayLengthTest()
-      : position_{0, 0, 0}, diagnostic_(spvDiagnosticCreate(&position_, "")) {}
+      : position_(), diagnostic_(spvDiagnosticCreate(&position_, "")) {}
```

Comment:
And reading the spec, TIL it will be zero-initialized.
8.5 
10:  - An object whose initializer is an empty set of parentheses, i.e., (), shall be value-initialized.
7: To value-initialize an object of type T means: .... if T is a (possibly cv-qualified) non-union class type without a user-provided constructor, then the object is zero-initialized and, if T’s implicitly-declared default constructor is non-trivial, that constructor is called.